src/dom-builder.ts

```diff
@@ -90,16 +96,17 @@ export type ClassDescription =
 export interface ClassDescriptionArray extends Array<ClassDescription> {}
 
 export type Attributes = {
-  [name: string]: (Showable | boolean) | Behavior<Showable | boolean>;
+  [name: string]:
+    | (Showable | boolean)
+    | Stream<Showable | boolean>
+    | Behavior<Showable | boolean>;
 };
 
 type _InitialProperties = {
   streams?: StreamDescriptions;
   behaviors?: BehaviorDescriptions;
   style?: Style;
-  props?: {
-    [name: string]: Showable | Behavior<Showable | boolean>;
-  };
+  props?: Attributes;
```

Comment:
This is not the same, since attributes are HTML attributes and not the properties on the DOMElement.
Since an attribute could be set in the HTML without any value, we need the boolean, however this is not the case for properties on the DOMElement.

Reply:
Ah, I didn't notice that changing it added `boolean` to `props`. But, I've looked it up [the documentation here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) and there are actually a couple of boolean-valued properties: `spellcheck`, `translate`, `draggable`, `hidden`, and more. So maybe this change is actually ok after all?